### PR TITLE
add tooltips for header

### PR DESCRIPTION
**Pull request by Tolik170**
**Added Tooltips for header**

image

### CODE REVIEW

File: src/containers/navigation-icons/student-icons/StudentIcons.js [hunk 1]
```diff
@@ -9,26 +11,45 @@ import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined
...
-        <MenuIcon color='primary' />
-      </IconButton>
+      <Tooltip arrow title={ t('iconsTooltip.language') }>
+        <IconButton size='large' sx={ style.langIcon }>
+          <LanguageIcon color='primary' />
+        </IconButton>
+      </Tooltip>
+
+      <Tooltip arrow title={ t('iconsTooltip.messages') }>
+        <IconButton sx={ style.sudentIcons }>
```
Comment (rostyslaw27) [inline src/containers/navigation-icons/student-icons/StudentIcons.js:25]: sudentIcons?

File: src/constants/translations/en/iconsTooltip.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "language": "Language",
+    "messages": "Messages",
+    "favorites": "Favorites",
+    "notifications": "Notifications",
+    "account": "Account",
+    "menu": "Menu",
+    "login": "Login"
+}
```
Comment (High-Voltaged) [inline src/constants/translations/en/iconsTooltip.json:9]: Please add end of line (EOL) character here.

**Review by High-Voltaged — CHANGES_REQUESTED** [review]

File: src/containers/navigation-icons/student-icons/StudentIcons.js (hunk 1 above)
Comment (Tolik170) [reply src/containers/navigation-icons/student-icons/StudentIcons.js]: fixed.

**Review by rostyslaw27 — APPROVED** [review]

File: src/constants/translations/en/iconsTooltip.json (hunk 2 above)
Comment (Tolik170) [reply src/constants/translations/en/iconsTooltip.json]: fixed.

**Review by maarekua — APPROVED** [review]

**Review by High-Voltaged — APPROVED** [review]

**Pull request merged**